nextflow.config

```diff
@@ -1,170 +1,308 @@
 /*
- * -------------------------------------------------
- *  nf-core/coproid Nextflow config file
- * -------------------------------------------------
- * Default config options for all environments.
- */
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    nf-core/coproid Nextflow config file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Default config options for all compute environments
+----------------------------------------------------------------------------------------
+*/
+
 // Global default params, used in configs
 params {
-  // Workflow flags
-  reads = ''
-  single_end = false
-  outdir = './results'
-  phred = 33
-  adna = true
-  genome1 = ''
-  genome2 = ''
-  genome3 = ''
-  name1 = ''
-  name2 = ''
-  name3 = ''
-  index1 = ''
-  index2 = ''
-  index3 = ''
-  fasta1 = ''
-  fasta2 = ''
-  fasta3 = ''
-  collapse = true
-  identity = 0.95
-  pmdscore = 3
-  library = 'classic'
-  bowtie = 'very-sensitive'
-  krakendb =''
-  minKraken = 50
-  endo1 = 0.01
-  endo2 = 0.01
-  endo3 = 0.01
-  sp_labels = "$baseDir/data/sourcepredict/modern_gut_microbiomes_labels.csv"
-  sp_sources = "$baseDir/data/sourcepredict/modern_gut_microbiomes_sources.csv"
-  sp_norm = 'gmpr'
-  sp_dim = 2
-  sp_embed = 'mds'
-  sp_neighbors = 'all'
-  name = false
-  multiqc_config = "$baseDir/assets/multiqc_config.yaml"
-  email = false
-  email_on_fail = false
-  max_multiqc_email_size = 25.MB
-  plaintext_email = false
-  monochrome_logs = false
-  help = false
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
-  tracedir = "${params.outdir}/pipeline_info"
-  igenomes_ignore = false
-  custom_config_version = 'master'
-  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
-  hostnames = false
-  config_profile_description = false
-  config_profile_contact = false
-  config_profile_url = false
-
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
-  
+
+    // Input options
+    input                      = null
+    // References
+    genome_sheet               = null
+    igenomes_base              = 's3://ngi-igenomes/igenomes/'
+    igenomes_ignore            = true
+
+    file_prefix                = 'coproid'
+    sam2lca_db                 = null
+    sam2lca_identity           = 0.9
+    sam2lca_acc2tax            = 'adnamap'
+    kraken2_db                 = null
+    sp_sources                 = null
+    sp_labels                  = null
+    taxa_sqlite                = "params.modules_testdata_base_path + /genomics/prokaryotes/metagenome/taxonomy/misc/taxa_sqlite.xz"
+    taxa_sqlite_traverse_pkl   = "params.modules_testdata_base_path + /genomics/prokaryotes/metagenome/taxonomy/misc/taxa_sqlite_traverse.pkl"
```

Comment:
This doesn't seem right, I think you missplaced a quote. The `modules_testdata_base_path` won't be used here